src/eip4824.ts

```diff
@@ -34,31 +34,84 @@ router.get('/:space', async (req, res) => {
 });
 
 router.get('/:space/members', async (req, res) => {
-  let space: any = {};
-
   try {
-    space = await getSpace(req.params.space);
+    const spaceId = req.params.space;
+    const cursor = req.query.cursor || null;  
+    const pageSize = 500; // Default page size
+
+    const space = await getSpace(spaceId);
+    if (!space.verified) {
+      return res.status(400).json({
+        error: 'INVALID_SPACE',
+        message: 'The specified space is not verified.'
+      });
+    }
 
-    if (!space.verified) return res.status(400).json({ error: 'INVALID' });
-  } catch (e) {
-    return res.status(404).json({ error: 'NOT_FOUND' });
-  }
+    let members: { type: string, id: string }[] = []; 
+    let nextCursor: string | null = null; 
+
+    if (cursor) {
+      const combinedMembersResult = await getCombinedMembersAndVoters(
+        spaceId,
+        cursor,
+        pageSize,
+        [], 
+        [], 
+        []  
+      );
+      members = combinedMembersResult.members.map(voter => ({ type: 'EthereumAddress', id: voter }));
```

Comment:
lint seems off, could you please run `yarn lint`

Reply:
Lint done